src/main/scala/mesosphere/marathon/util/RoleSettings.scala

```diff
@@ -34,7 +34,7 @@ object RoleSettings extends StrictLogging {
       // We have a service in the root group, no enforced role here
       RoleSettings(validRoles = Set(defaultRole), defaultRole = defaultRole)
     } else {
-      val topLevelGroupPath = servicePathId.rootPath
+      val topLevelGroupPath = servicePathId.rootPath.asAbsolutePath
```

Comment:
AbsolutePath.rootPath should return an absolute path as well, i guess? There should be no need to call asAbsolutePath again

Reply:
Yes. `asAbsolutePath` is deprecated anyways. However, in some cases it was just simpler for now. I did not want to blow up this diff.